[runner] Print prompt before call into text prefiller

larryliu0820 opened the pull request:
Summary: Currently we are printing out prompt token by token in `TextPrefiller.prefill()`. This is unnecessary and slow things down. Moving it out of text prefiller and print the prompt in runner.

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

kimishpatel reviewed (commented):
There might be some internal call sites relying on the callback

larryliu0820 committed:
Print prompt before call into text prefiller (#5019)

Summary:
Currently we are printing out prompt token by token in `TextPrefiller.prefill()`. This is unnecessary and slow things down. Moving it out of text prefiller and print the prompt in runner.


Reviewed By: kimishpatel

Differential Revision: D62053820

Pulled By: larryliu0820
